Patch-Source: https://salsa.debian.org/utopia-team/avahi/-/blob/b1a3a539ac0b6d79a7a0b87dccbb99028e10ec63/debian/patches/Fetch-build-db-from-upstream-git.patch
this is needed when building python
--
From: Andreas Henriksson <andreas@fatal.se>
Date: Sat, 26 Oct 2019 15:49:35 +0200
Subject: Fetch build-db from upstream git

Fetch the build-db file from:
https://raw.githubusercontent.com/lathiat/avahi/master/service-type-database/build-db

This file is missing dist tarball, likely caused by changes in:
https://github.com/lathiat/avahi/pull/134

Carry it as a patch for now.
---
 service-type-database/build-db | 49 ++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 49 insertions(+)
 create mode 100755 service-type-database/build-db

diff --git a/service-type-database/build-db b/service-type-database/build-db
new file mode 100755
index 0000000..78ee892
--- /dev/null
+++ b/service-type-database/build-db
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*-python-*-
+# This file is part of avahi.
+#
+# avahi is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# avahi is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with avahi; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307
+# USA.
+
+try:
+    import anydbm as dbm
+except ImportError:
+    import dbm
+
+import sys
+
+if len(sys.argv) > 1:
+    infn = sys.argv[1]
+else:
+    infn = "service-types"
+
+if len(sys.argv) > 2:
+    outfn = sys.argv[2]
+else:
+    outfn = infn + ".db"
+
+db = dbm.open(outfn, "n")
+
+for ln in open(infn, "r"):
+    ln = ln.strip(" \r\n\t")
+    
+    if ln == "" or ln.startswith("#"):
+        continue
+
+    t, n = ln.split(":", 1)
+
+    db[t.strip()] = n.strip()
+
+db.close()
